src/styles/utils/mods.scss

```diff
@@ -0,0 +1,23 @@
+.box-1 {
+  background-color: red;
+}
+
+.box-2 {
+  background-color: rgb(204, 0, 0);
+}
+
+.box-3 {
+  background-color: rgb(153, 0, 0);
+}
+
+.box-4 {
+  background-color: rgb(102, 0, 0);
+}
+
+.box-5 {
+  background-color: rgb(61, 0, 0);
+}
+
+.box-6 {
+  background-color: black;
+}
```

Comment:
You should be able to do this with an @each loop. Even though it's going to look like this in CSS anyway, I think mentors may want you to assign the colors using an @each loop. (Especially now that someone suggested this. :))))